client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -150,3 +150,37 @@ export const NegativeBalanceDepositsPausedNotice: React.FC = () => (
 		} ) }
 	</InlineNotice>
 );
+
+/**
+ * Renders a notice informing the user that deposits only occur when there are funds available.
+ */
+export const NoFundsAvailableForDepositNotice: React.FC = () => (
+	<InlineNotice
+		status="warning"
+		icon
+		className="no-funds-available-for-deposit-notice"
+		isDismissible={ false }
+	>
+		{ interpolateComponents( {
+			mixedString: sprintf(
+				/* translators: %s: WooPayments */
+				__(
+					'You have no funds available to deposit. {{whyLink}}Why?{{/whyLink}}',
+					'woocommerce-payments'
+				),
+				'WooPayments'
+			),
```

Comment:
Suggestion: remove redundant sprintf that isn't being used.


```suggestion
		{ interpolateComponents( {
			mixedString: __(
				'You have no funds available to deposit. {{whyLink}}Why?{{/whyLink}}',
				'woocommerce-payments'
			),
```